App/Components/VendorLocator.js

```diff
@@ -37,8 +37,13 @@ class VendorLocator extends React.Component {
   // }
 
   renderMapMarkers (vendor) {
+    const current_vendor = this.props.current_vendor;
+    let pinColor = 'red';
+    if (current_vendor && vendor.name === current_vendor.name) {
```

Comment:
It feels like there should be a better way to determine the vendor.

Probably their id that we should be pulling from the server